api/koko_api.go

```diff
@@ -620,9 +628,11 @@ func MakeVeth(veth1 VEth, veth2 VEth) error {
 
 	if err = veth1.SetVethLink(link1); err != nil {
 		netlink.LinkDel(link1)
+		netlink.LinkDel(link2)
```

Comment:
We don't need this because veth must be exists with pairs of veth. So if one is removed then another must be removed. Is there any clear corner cases which one veth is still remained?

Reply:
For example, an existing interface with the same link name will cause `netlink.LinkSetName` to fail with `file exists` error, in this case the link with `tempLinkName1` remains in the container. Anyway, this should be in a new PR if needed.